src/components/ProductCard/ProductCard.css

```diff
@@ -0,0 +1,53 @@
+.product {
+  width: 23%;
```

Comment:
try to use absolute width for a product card.

Reply:
will fix in the next pr